docs/configuration.md

```diff
@@ -36,15 +36,19 @@ You shall not share the private key (or the password) with anyone.
 
 The instructions below assume that:
 * The node is deployed in directory `/tmp/node`;
-* The initial network peer to join is "http://31.13.249.1:3013/".
+* The initial network peers to join are "aenode://pp$2eDAWTgveKp1C4dWhy9Hg59NCrg8TPUCKSXeEgvnPdro4ra177@31.13.249.1:3015",
```

Comment:
This looks conflicting with list in RELEASE-NOTES-0.10.0.md ? cc @dincho

Reply:
Which list? The one that list some HTTP addresses for inspection? That list serves another purpose I think...

Reply:
> The one that list some HTTP addresses for inspection?

Yes that one. Though impact is similar: public IP addresses users expect to be up. Cc @dincho